.dockerignore

```
@@ -0,0 +1,3 @@
+target/
+.github/
+.idea/
```

Add a newline att end of file.
```suggestion
.idea/

```

---

This change was already added